client/src/components/ReportsByTask.js

```diff
@@ -79,10 +81,10 @@ export default class ReportsByPoam extends Component {
     let titleSuffix = ''
     let resetFnc = ''
     let resetButtonLabel = ''
-    if (this.state.focusedPoam) {
-      titleSuffix = `for ${this.state.focusedPoam.shortName}`
-      resetFnc = 'goToPoam'
-      resetButtonLabel = 'All PoAMs'
+    if (this.state.focusedTask) {
+      titleSuffix = `for ${this.state.focusedTask.shortName}`
+      resetFnc = 'goToTask'
+      resetButtonLabel = 'All tasks'
```

Comment:
And maybe here also use the plural version for taskShortTitle.

Reply:
resetFnc is a callback function, thus should not be linked to the dictionary